src/commands/session/createSessionCommand.ts

```diff
@@ -0,0 +1,23 @@
+import { CreateSessionInterface } from "../../interfaces/session";
+
+export default class CreateSessionCommand {
+    private readonly _sessionToken: string;
```

Comment:
same here, let's just create an interface and pass an object around, like authorizationCommand